common/djangoapps/dark_lang/middleware.py

```diff
@@ -158,8 +129,3 @@ def _activate_preview_language(self, request):
 
```

Comment:
I am not sure which method your are pointing to, but I will apply this in multiple locations if it seems possible.  I have no problem making this change.